src/main/scala/com/nec/spark/agile/SparkExpressionToCExpression.scala

```diff
@@ -486,6 +500,9 @@ object SparkExpressionToCExpression {
       case VeScalarType.VeNullableInt    => IntegerType
       case VeScalarType.VeNullableLong   => LongType
       case VeString                      => StringType
+      case _ => sys.error(s"SparkExpressionToCExpression.likelySparkType: " +
```

Comment:
This is also covered by pattern matching, never going to reach.

Reply:
I'm surprised SonarCloud doesn't catch that it's unnecessary.  I thought it would give us a warning for that.

Reply:
I had to convince myself it was safe to remove this check. Yeah, it seems unlikely the set of types would change.